src/components/wallet/card/wallet-card.tsx

```diff
@@ -165,8 +166,15 @@ const WalletCard = ({
         );
     };
 
+    const getSvgWithUniqueIds = (code: string) => {
+        // A workaround to ensure that important ids in svg <defs> are unique to avoid issues with rendering in Storybook Docs:
+        const unique_id = v4();
+        return code.replaceAll(/id="/g, `id="${unique_id}`).replaceAll(/url\(#/g, `url(#${unique_id}`);
```

Comment:
```suggestion
        return code.replace(/id="/g, `id="${unique_id}`).replace(/url\(#/g, `url(#${unique_id}`);
```
Since we have the global regex `/g`, it will replace every occurrence. No need to use `replaceAll`

Reply:
@balakrishna-binary i need to replace all ids with unique ones 🙂

Reply:
@balakrishna-binary thank you so much! applied your suggestion 🙂